e2e test startup failure coming from unrelated issue:

======= Adding new dynamic plugin configuration for @backstage-community/plugin-todo
Traceback (most recent call last):
  File "/opt/app-root/src/install-dynamic-plugins.py", line 1556, in <module>
    main()
  File "/opt/app-root/src/install-dynamic-plugins.py", line 1510, in main
    merge_plugin(plugin, all_plugins, dynamic_plugins_file, level=1)
  File "/opt/app-root/src/install-dynamic-plugins.py", line 151, in merge_plugin
    return OciPackageMerger(plugin, dynamic_plugins_file, all_plugins).merge_plugin(level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/app-root/src/install-dynamic-plugins.py", line 611, in merge_plugin
    raise InstallException(
InstallException: Cannot use {{inherit}} for oci://registry.access.redhat.com/rhdh/red-hat-developer-hub-backstage-plugin-lightspeed: no existing plugin configuration found. Ensure a plugin from this image is defined in an included file with an explicit version.

--- a/dynamic-plugins/package.json
+++ b/dynamic-plugins/package.json
@@ -37,8 +37,7 @@
     "@types/react-dom": "18.3.7",
     "refractor@npm:3.6.0/prismjs": "^1.30.0",
     "infinispan": "0.13.0",
-    "react-use": "17.6.1",
-    "@segment/analytics-next@npm:1.81.1/js-cookie": "3.0.8"
+    "react-use": "17.6.1"
   },
   "packageManager": "yarn@4.12.0"
 }

